phabfive/cli.py

```diff
@@ -135,7 +135,6 @@ def parse_cli():
             base_args,
             options_first=True,
             version=phabfive.__version__,
-            help=True,
```

Comment:
So this would change the core behaviour of how docopt works? Could you elaborate why we don't need this one any longer?

Reply:
This is more about reducing/minimizing the impact on systems installed with docopt-ng instead of docopt.

Per [When docopt and docopt-ng are in the same venv, docopt takes precidence](https://github.com/jazzband/docopt-ng/issues/22), the maintainer of docopt-ng is considering a PEP 541 to take over docopt and by then, line 138 of cli.py will break due to the breaking change introduced by docopt-ng: 
in docopt, the variable is `docopt(help=boolean) `
https://github.com/jazzband/docopt-ng/blob/master/docopt/__init__.py line 865
while docopt-ng is `docopt(default_help=boolean)`
https://github.com/docopt/docopt/blob/master/docopt.py line 505

I would suggest either
1) remove `help=True` (as is) and accept the side effect of no more showing help message by default
2) use docopt.__version__ to decide which to import additional module, [packaging](https://packaging.python.org/en/latest/)

```
import doctopt
from packaging import version

if (version.parse(docopt.__version__) >= version.parse("0.6.2"): # docopt is installed
        cli_args = docopt(
            base_args,
            options_first=True,
            version=phabfive.__version__,
            help=True,
        )
else: # docopt-ng is installed
        cli_args = docopt(
            base_args,
            options_first=True,
            version=phabfive.__version__,
            default_help=True,
        )
```

Please let me know which approach would be preferred?

Reply:
ohhh neat, did not know anyone had finally forked out docopt and made a version that was more widely used and got traction on it

So in either case, as long as the help message behaves the same for the toplevel command and for the sublevel commands, then either solution is find really. I think we had some special case with how we parse sub-commands, but i think the `extras(...)` call deals with that. Test out that commands do not pass through unintentionally to the sub-parsers if you simply run for example `phabfive` phabfive maniphest` etc and that correct parser instances deals with them

Reply:
I have actually ran using both docopt and docopt-ng but not necessary testing exhaustively to be 100% confident that I covered 100%.... will try 'phabfive phabfive user whoam i' nonetheless.